qiskit/synthesis/two_qubit/two_qubit_decompose.py

```diff
@@ -104,8 +110,7 @@ class TwoQubitWeylDecomposition:
 
     .. math::
 
-        U \in U(4),~
-        {K_1}^l, {K_1}^r, {K_2}^l, {K_2}^r \in SU(2)
+        U \in U(4),\sim {K_1}^l, {K_1}^r, {K_2}^l, {K_2}^r \in SU(2)
```

Comment:
here it shouldn't be sin. the tilde is only to make some space

Reply:
can you fix the lint errors, so that we can go over the final docs generated by CI?